utils/githooks/find_base.sh

```diff
@@ -6,7 +6,7 @@ if ! BRANCH=$(git rev-parse --abbrev-ref HEAD 2>/dev/null); then
 fi
 
 ORIGIN="${DAOS_ORIGIN:=origin}"
-if [ "$ORIGIN" == "origin" ]; then
+if [ "$ORIGIN" = "origin" ]; then
```

Comment:
Is this a deliberate change?

Reply:
@daltonbohning perhaps you could add this to your change?

Reply:
Ah! I meant to do that and forgot. If I need to push to #13135 again I will. Otherwise I can just push separately if this PR isn't ready soon?